### DISCUSSION TITLE
Can api select fields from item based on Conditions used in UI?

### DISCUSSION THREAD

**Discussion by Slembit** [directus/directus#16277] [Archive]
In the item editor you can dynamically hide or set fields as read only based on other field value in same item. Sweet!
But is there a way to extend/mirror this behavior to the API also?

I mean instead of deselecting the fields manually, can the API return everything except those fields that did match the dynamic condition?

or at least from api return the field/fields and its metadata about condition so I can apply those filters myself?

![condition](https://user-images.githubusercontent.com/13945464/199501697-19f7066f-bdd6-4214-80e6-43b684923dfd.png)

**Comment by rijkvanzanten** [comment]
The API doesn't execute those conditions, they're purely visual/ UX for the app
